Index: CT.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- CT.py	(date 1519674750117)
+++ CT.py	(date 1519674750117)
@@ -10,8 +10,11 @@
 import os
 
 vse=np.load('C:/MEDSLIKE/numpy/surface.17/vse.npy') #slike
-povp = np.mean(vse, 0)
+povp = np.mean(vse,0)
 stdd = np.std(vse,0)
+#povp = np.mean(vse, 0)
+#stdd = np.std(vse,0)       STARO
+
 
 kvse = np.load('C:/MEDSLIKE/numpy/xyzTUMORJAzaPRVIH300slik.npy') #koordinate
 kpovp = np.mean(kvse,0)
@@ -128,12 +131,6 @@
 
 
 
-def getBatch(size, min, max):
-
-
-
-    return bla
-
 
 #OLD!!:
 def standardize2D(arr):
@@ -141,7 +138,7 @@
 def standardizecoords(arr):
     return np.nan_to_num((arr-kpovp)/(kstdd+0.00001))
 
-def getBatchOld(size = 10, maxsize = 299, minsize = 0):   #return arr, coordinates
+def getBatch(size = 10, maxsize = 299, minsize = 0):   #return arr, coordinates
     rn = np.random.randint(0, maxsize+1-minsize, size = (size))
     arr = []
     coordinates = []
@@ -153,7 +150,7 @@
 
     return arr, coordinates
 
-def getBatchTestOld(size = 10, min = 299, max = 335):
+def getBatchTest(size = 10, min = 299, max = 335):
     arr = []
     coordinates = []
     for i in np.random.randint(min,max, size = size):
